Retry check processing time (#921)
ASFHyP3/hyp3: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [2.14.2]
### Fixed

> - Step function now retries errors when submitting checking AWS Batch processing time.

Reviewer: ```suggestion
- Step function now retries the CHECK_PROCESSING_TIME task when errors are encountered.
```
